src/lib/containers/Error.tsx

```diff
@@ -21,12 +25,20 @@ export const Error = (props: ErrorProps) => {
         variant={'danger'}
         transition={'div'}
       >
-        {error.status === 403 && (
-          <p>
-            Please <SignInButton /> to view this resource.
-          </p>
-        )}
-        <p>{error.reason}</p>
+        <p>
+          {loginError && (
+            <>
+              Please <SignInButton /> to view this resource.
```

Comment:
Will `<SignInButton />` cause re-mounting problems?

Reply:
It shouldn't, and if it did re-mount a bunch of extra times it would be less of an issue since it doesn't make any API calls, but thank you for pointing out!